README.adoc

```diff
@@ -411,10 +422,6 @@ image::https://user-images.githubusercontent.com/7708031/53174794-8878a600-35e2-
 * https://golang.org/[Golang] - Programming language used
 * https://echo.labstack.com/[Echo] - Web framework used
 
-== Contributing
```

Comment:
@camilamacedo86 I think we should keep this section in and remove the line about contributing from this section: https://github.com/aerogear/mobile-security-service#prerequisites

Reply:
👍 catcher . Done.